### PR TITLE
Update description of TimeEval result concept page

### PR DESCRIPTION
**Pull request by mshubham00**

### CODE REVIEW

File: docs/concepts/results.md [hunk 1]
```diff
@@ -1,7 +1,89 @@
 # TimeEval results
 
-How to work with the results of TimeEval.
+Upon executing TimeEval python script for the given datasets, timeeval checks the effectiveness of selected algorithm and generate an output file as `./TimeEval/results/<timestamp>` directory. The output directory has following data structure:
+
+```bash
+results/<time-stamp>/
+├── results.csv
+├── algorithm_1/hyperparameter/
+|   ├── dataset_1/epoch/
```
Comment (CodeLionX) [inline docs/concepts/results.md:9]:
```suggestion
├── <algorithm_1>/<hyper_params_id>/
|   ├── <collection_name>/<dataset_name_1>/<repetition_number>/
```
Please change the following lines accordingly

File: docs/concepts/results.md [hunk 2]
```diff
@@ -1,7 +1,89 @@
...
+│   |   ├── anomaly_scores.ts
+│   |   ├── docker-algorithm-scores.csv
+│   |   ├── execution.log
+│   |   ├── hyper_params.json
+│   |   └── metrics.csv
+|   └── dataset_2/epoch/
+│       ├── anomaly_scores.ts
+│       ├── docker-algorithm-scores.csv
+│       ├── execution.log
+|	├── model.pkl
```
Comment (CodeLionX) [inline docs/concepts/results.md:19]: Please don't use tabs, always use spaces for indentation.

File: docs/concepts/results.md [hunk 3]
```diff
@@ -1,7 +1,89 @@
 # TimeEval results
 
-How to work with the results of TimeEval.
+Upon executing TimeEval python script for the given datasets, timeeval checks the effectiveness of selected algorithm and generate an output file as `./TimeEval/results/<timestamp>` directory. The output directory has following data structure:
```
Comment (CodeLionX) [inline docs/concepts/results.md:3]: Please improve the spelling and writing throughout the document; e. g.:
```suggestion
Upon configuring and executing TimeEval, TimeEval executes the algorithms with their configured hyperparameter values on all the datasets.
It measures the algorithms' runtimes and checks their effectiveness using evaluation measures (:doc:`metrics <api/timeeval.metrics>`).
The results are stored in a summary file and a nested folder structure in a results folder (``./results/<timestamp>`` per default).
The output directory has the following structure:
```

File: docs/concepts/results.md [hunk 4]
```diff
@@ -1,7 +1,89 @@
...
+        ├── hyper_params.json
+        └── metrics.csv
 
-```{warning}
-WIP
 ```
+Description of each file is given below. 
+
+## result.csv
+The file `result.csv` contains following attributes:
```
Comment (CodeLionX) [inline docs/concepts/results.md:34]: Please always add empty lines after headings.

File: docs/concepts/results.md [hunk 5]
```diff
@@ -1,7 +1,89 @@
...
+| execute_preprocess_time| float64 | |
+| execute_main_time | float64 | time taken for execute the script for the given datafile |
+| execute_postprocess_time|  float64 | |
+| status| str | assigns flag value upon the process |
+| error_message| str | |
+| repetition| int | number of times dataset being processed |
+| hyper_params| float64 | for given algorithm |
+| hyper_params_id| float64 | 32 alphanumerical characters |
+| metric_1| float64 | --user-- |
+| metric_2|  float64 | --user-- |
```
Comment (CodeLionX) [inline docs/concepts/results.md:56]: - algorithm: name of the algorithm as defined in :class:`~timeeval.Algorithm`'s `name` attribute
- collection: name of the dataset collection. A collection groups similar datasets.
- dataset: name of the dataset
- dataset_training_type: specifies, whether a dataset has a training time series with anomaly labels (supervised), with normal data only (semi-supervised), or no training time series at all (unsupervised)
- dataset_input_dimensionality: specifies if the dataset has multiple channels (multivariate) or not (univariate)
- algo_training_type: specifies, whether an algorithm requires training data with anomalies (supervised), without normal data only (semi-supervised), or does not require training data (unsupervised)
- algo_input_dimensionality: univariate or multivariate (see above)
- train_preprocess_time: runtime of the preprocessing step during training in seconds
- train_main_time: runtime of the training in seconds (does not include pre-processing time)
- execute_preprocess_time: runtime of the preprocessing step during execution in seconds
- execute_main_time: runtime of the execution of the algorithm on the test time series in seconds (does not include pre- or post-processing times)
- execute_postprocess_time: runtime of the post-processing step during execution
- status: specifies, whether the algorithm executed successfully (:obj:`~timeeval.Status.OK`), exceeded the time limit (:obj:`~timeeval.Status.TIMEOUT`), exceeded the memory limit (:obj:`~timeeval.Status.OOM`), or failed (:obj:`~timeeval.Status.ERROR`)
- error_message: optional detailed error message
- repetition: repetition number if a dataset-hyperparameter-dataset combination was executed multiple times
- hyper_params: actual hyperparameter values for this execution
- hyper_params_id: alphanumerical hash of the hyperparameter configuration
- metric_1: value of the first performance metric
- ...

File: docs/concepts/results.md [hunk 6]
```diff
@@ -1,7 +1,89 @@
...
+
+Result associated with respective dataset and algorithm is stored in following files:
+
+### anomaly_scores.ts
+
+### docker-algorithm-scores.csv
+
+### model.pkl
+
+### execution.log
```
Comment (CodeLionX) [inline docs/concepts/results.md:69]: Please add descriptions

File: docs/concepts/results.md [hunk 7]
```diff
@@ -1,7 +1,89 @@
...
+   - Recall
+   - F1Score
+2. AUC-metrics: for continuous dataset
+   - RocAUC
+   - PrAUC
+4. Range-metrics: analyse segment of the dataset
+   - RangePrecision
+   - RangeRecall
+   - RangeFScore
+   - RangePrecisionRangeRecallAUC
```
Comment (CodeLionX) [inline docs/concepts/results.md:84]: Please replace with a brief description and a link to the `api/timeeval.metrics`-section in the documentation.
Add a note that this file also always includes the runtime measurements.

**Review by CodeLionX — CHANGES_REQUESTED** [review]

File: docs/concepts/results.md (hunk 2 above)
Comment (mshubham00) [reply docs/concepts/results.md:19]: i will keep in mind

**Review by CodeLionX — APPROVED** [review]

**Pull request merged**